core/src/avm1/globals/system_capabilities.rs

```diff
@@ -7,12 +7,13 @@ use crate::avm1::{ScriptObject, Value};
 use crate::string::AvmString;
 use gc_arena::MutationContext;
 
+// Variables have been added in later flash player versions.
+// To emulate exact flash player behaviour, properties can only be returned
+// if the flash player version is at least the one in which they were added.
+// Whether the properties are returned is independent of the SWF version.
 const OBJECT_DECLS: &[Declaration] = declare_properties! {
```

Comment:
TODO for a future PR, is that I believe these are all vanilla fields and not properties (e.g. `System.capabilities.avHardwareDisable = "foobar";` works.)

Reply:
Since I didn’t stub this out, I’m not sure whether this is just a note for the future or I should change something about that comment. What did you mean?

Reply:
No change necessary because as you necessary, this was an issue already prior to your PR -- just making a note of it for myself in case I forget later.